api/openapi/openapi_metadata.py

```diff
@@ -0,0 +1,31 @@
+openapi_metadata = {
+    "title": "Observations Europe EUMETNET",
+    "description": (
+        "EDR data service for European environmental observations from EUMETNET," " co-funded by the European Union"
+    ),
+    "contacts": [
+        {
+            "organization": "EUMETNET",
+            "addresses": [
+                {
+                    "deliveryPoint": ["Avenue Circulaire 3"],
+                    "city": "Bruxelles",
+                    "postalCode": "1180",
+                    "country": "Belgique",
+                }
+            ],
+            "links": [
+                {
+                    "rel": "about",
+                    "type": "text/html",
+                    "href": "https://www.eumetnet.eu/about-us/",
+                }
+            ],
+            "roles": ["host"],
+        },
+    ],
```

Comment:
What use is a postal address to the user of this API?
Also, in the landingpage we put a different address (rodeoproject@fmi.fi), which is probably much more useful to the users...